### DISCUSSION TITLE
Ways to receive Selling Partner API documentation updates

### DISCUSSION THREAD

**Discussion by vaidas-lungis** [amzn/selling-partner-api-models#3654] [General] [answered]
is there are a way to get https://developer-docs.amazon.com/sp-api/docs page updates via any format or service? I would like to get page updates whenever it changes. It would be awesome to have a diff too.

Lately, I seen a lot of changes across the documents and it is hard to follow the updates in [announcements](https://github.com/amzn/selling-partner-api-docs/discussions/categories/announcements), Moreover it is a summary ( filtered information) of the changes. The RSS feed is a reflection of the announcements? 

The previous way when everything was on github, it allowed to be on the update pulse. Now

**Comment by ritikacd** [comment, marked as answer]
Hi @vaidas-lungis,

We recommend subscribing to RSS feeds to receive updates on product feature releases and blogs. Yes, RSS feed reflects announcements. Currently, we do not support documentation page updates via RSS feed. However, we can submit the feature request to the team.

You can also integrate RSS feed with the slack, so that whenever there's an update you should be notified realtime on the slack. Please refer to this video to [integrating SP-API RSS feeds into Slack](https://www.youtube.com/watch?v=WT9miRiMA2g).

Thanks,
Ritika Chaudhary
Selling Partner API Team
